docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>I'd give a few words on what is it. something like:</p><p> </p><h1>Alert Activations</h1><p> </p><p>When an existing alert[link to alerts page] is activated by its configured trigger, MLRun saves the activation record.</p><p> </p><h2>Listing alerts activations</h2><p>This can be done in three ways:</p><p> </p><ol><li>Getting activations using MLRun's Run db</li><li>Getting activations for a specific project</li><li>Getting activations for an alert config object</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>